frontend/src/Components/Login/SignUp1.jsx

```diff
@@ -0,0 +1,69 @@
+import { Navigate,useNavigate } from "react-router"
```

Comment:
There are 2 files for Signup. Clean up the one which is not being used.

Reply:
done